### DISCUSSION TITLE
is it possible to show margin for the epub file?

### DISCUSSION THREAD

**Discussion by TripleStrike** [sumatrapdfreader/sumatrapdf#3137] [Q&A] [answered]
I just notice that sumatrapdf open an epub and all the text in the book just close to the book page border.Tthere is no margin left. I haven't made any changes to the configuration.

are there any way to add some space to the margin I feel the real book should have some margin at the border of the book rather than no space at all.

thank you.
(Tried to search with "epub margin" and can't find a solution)

**Comment by GitHubRulesOK** [comment, marked as answer]
Versions may differ but there should be a margin in any fixed page whereas reflow should not have fixed margin but should still have one
![image](https://user-images.githubusercontent.com/12754913/204028138-d85a80a5-8448-4abe-a44f-ab646c678482.png)
![image](https://user-images.githubusercontent.com/12754913/204028488-45e247aa-564e-489a-b550-f5dc840e697d.png)
![image](https://user-images.githubusercontent.com/12754913/204028705-8c9cc2d5-746e-4f26-a65b-895b02923980.png)

**Comment by TripleStrike** [comment]
Thank you I think this issue is the epub itself does not have margin. it's not the SumatraPDF issue.
